src/amqp-writable-stream.js

```diff
@@ -0,0 +1,14 @@
+const { Writable } = require('stream');
+const stringify = require('json-stringify-safe');
+
+module.exports = class AmqpWritableStream extends Writable {
+  constructor (channel, queueName, opts = {}) {
+    super(Object.assign(opts, { objectMode: true }));
+    this.channel = channel;
+    this.queueName = queueName;
+    channel.on('drain', () => this.emit('drain'));
+  }
+  write (chunk, encoding, callback) {
+    return this.channel.sendToQueue(this.queueName, Buffer.from(stringify(chunk)));
+  }
```

Comment:
Maybe some comments about how the backpressure works here would be good.  Does the `drain` event cause the backpressure to go away, coupled with the fact that we don't call `callback`?